insights/specs/default.py

```diff
@@ -448,7 +448,8 @@ def httpd_cmd(broker):
     ipaupgrade_log = simple_file("/var/log/ipaupgrade.log")
     ipcs_m = simple_command("/usr/bin/ipcs -m")
     ipcs_m_p = simple_command("/usr/bin/ipcs -m -p")
-    ipcs_s = simple_command("/usr/bin/ipcs -s")
+    ipcs_s = ipcs.LocalSpecs.ipcs_s_cmd
```

Comment:
@huali027 this spec can be left in the original form as a `simple_command`.  There's no need to create the LocalSpec in the datasource.